src/Microsoft.DotNet.Build.Tasks/PackageFiles/init-tools.cmd

```diff
@@ -26,7 +27,8 @@ if [%BUILDTOOLS_USE_CSPROJ%]==[] (
 { ^
   "dependencies": ^
     { ^
-      "MicroBuild.Core": "%MICROBUILD_VERSION%" ^
+      "MicroBuild.Core": "%MICROBUILD_VERSION%", ^
+      "Microsoft.Net.Compilers": "%ROSLYNCOMPILERS_VERSION%" ^
```

Comment:
yeah, I haven't done those changes yet, this PR only got updated because I've been making changes on my local buildtools and keeping track of the changes by pushing but I'm mainly getting corefx to build with the changes first, After getting that, I will update all of this logic to be msbuild based instead.